pos_cash_move_reason/tests/test_pos_cash_move_reason.py

```diff
@@ -13,8 +13,28 @@ def setUpClass(cls):
         cls.PosSession = cls.env["pos.session"]
         cls.WizardReason = cls.env["wizard.pos.move.reason"]
         cls.AccountMoveLine = cls.env["account.move.line"]
-
-        cls.config = cls.env.ref("point_of_sale.pos_config_main").copy()
+        cls.pricelist = cls.env["product.pricelist"].create(
+            {
+                "name": "Test pricelist",
+                "item_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "applied_on": "3_global",
+                            "compute_price": "formula",
+                            "base": "list_price",
+                        },
+                    )
+                ],
+            }
+        )
+        cls.config = cls.env.ref("point_of_sale.pos_config_main").copy(
+            {
+                "available_pricelist_ids": [(6, 0, cls.pricelist.ids)],
+                "pricelist_id": cls.pricelist.id,
+            }
+        )
```

Comment:
why not use core function ? 

https://github.com/odoo/odoo/blob/14.0/addons/point_of_sale/tests/common.py#L596

Reply:
I've given it a shot but it looks that class pulls too many stuff to just use it out of the box without much refactor. As we're moving to future versions with these customer, I'd let the research for `16.0`